### PR TITLE
denylist and allowlist

### PR DESCRIPTION
**Pull request by anjor**
**Goals (and why)**:
Use inclusive language.

**Implementation Description (bullets)**:

**Testing (What was existing testing like?  What have you done to improve it?)**:

**Concerns (what feedback would you like?)**:
Did I break something?
There are a few instances in the config, which will be trickier to handle.

**Where should we start reviewing?**:

**Priority (whenever / two weeks / yesterday)**:

### CODE REVIEW

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java [hunk 1]
```diff
@@ -91,12 +91,12 @@
     public CassandraService(
             MetricsManager metricsManager,
             CassandraKeyValueServiceConfig config,
-            Blacklist blacklist,
+            Denylist denylist,
```
Comment (Jolyon-S) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java:94]: Also a break, but this is only used internally to the best of my knowledge.

**Review by Jolyon-S — COMMENTED** [review]
There are two API breaks that I want you to ACK before merging, but I think both of these are niche enough not to be a problem here.

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Denylist.java [hunk 2]
```diff
@@ -33,38 +33,38 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class Blacklist {
-    private static final Logger log = LoggerFactory.getLogger(Blacklist.class);
+public class Denylist {
```
[the remark this answers is not in the document]
Comment (anjor) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Denylist.java:36]: Yeah, I am not sure how to guarantee its not used outside. Would defer to @jeremyk-91 / @gmaretic

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java (hunk 1 above)
Comment (anjor) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java:94]: Yeah, I am not sure how to guarantee its not used outside. Would defer to @jeremyk-91 / @gmaretic

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Denylist.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (CRogers) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Denylist.java:36]: (You can use sourcegraph to search for `com.palantir.atlasdb.keyvalue.cassandra.Blacklist` and you'll see that no one uses it, other than atlasdb clones/hackweek projects)

[the remark this answers is not in the document]
Comment (jeremyk-91) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Denylist.java:36]: How to do this is normally to check in Sourcegraph for this origin. In this case, the original `Blacklist` is not used.

[the remark this answers is not in the document]
Comment (jeremyk-91) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Denylist.java:36]: yeah, @CRogers ninja'd me 🏎️

**Pull request closed without merging**